docs/notebooks/diagnostics/line_integrated_diagnostic.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>My first impression when reading this whole heading was "Oh, I have to create a custom class to use this? I wonder if we could not make this easier to use somehow" - and then the last sentence up-ends my understanding! Perhaps we could reword the title to something like "Example: custom Interferometer class", and in the beginning of this part say something about how this is mostly a didactic example so people can wrap their head around the pre-made <code>Interferometer</code> and the whole LineIntegratedDiagnostic framework easier?</p><p> </p><p>It would also be helpful to point people to a place in the docs where they can find e.g. a schema for the arguments of <code>evaluate</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Ah, good point! I've added the following to the beginning of that section to hopefully head off the confusion: 

```
A [LineIntegratedDiagnostic] synthetic diagnostic for interferometry ([Interferometer]) is already implemented in PlasmaPy. However, in this notebook we will re-create this class in order to demonstrate how a custom subclass of [LineIntegratedDiagnostic] can be used to create a new synthetic diagnostic.
```

There should probably be a separate (short) notebook explaining the actual `Interferometer` class? That would hopefully also help dispel the confusion and provide a more natural place to document the `evaluate` method.

Reply:
It's probably fine to keep the example in this notebook. If/once it grows out, we can add another one.

Reply:
Ok, I'll keep it like that for now. Eventually there will be a bunch of these LineIntegratedDiagnostics, so then it may make sense for them each have a notebook?